packages/medusa/src/services/order-edit.ts

```diff
@@ -680,6 +680,39 @@ export default class OrderEditService extends TransactionBaseService {
     })
   }
 
+  async complete(
+    orderEditId: string,
+    context: { loggedInUserId?: string }
+  ): Promise<OrderEdit> {
+    return await this.atomicPhase_(async (manager) => {
```

Comment:
thought: would not have this method here but just call `.confirm` directly from the request handler

Reply:
I think it is handy to have a method that will check the status, payments and then confirm instead of having the consumer to do it. Also, it would avoid having business logic directly into the handler. wdyt?